lib/surface/ast.ex

```diff
@@ -45,11 +46,12 @@ defmodule Surface.AST.Expr do
       * `:value` - a quoted expression
       * `:meta` - compile meta
   """
-  defstruct [:value, :meta]
+  defstruct [:value, :meta, directives: []]
```

Comment:
The property `directives` under `## Properties` should be documented.

Same for all structs that newly accepts the property `directives`

Reply:
i think i got them all